### PR TITLE
{bio}[system/system] manta v1.6.0

### PR DESCRIPTION
**Pull request by robqiao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/manta/manta-1.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Software License::	GPLv3.0
+# Notes::		This is a binary build and only works on 64-bit linux
+##
+
+easyblock = 'CmdCp'
+
+name = 'manta'
+version = '1.6.0'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/manta/manta-1.6.0.eb]: should'nt it be `https://github.com/Illumina/manta`?

Comment (robqiao) [reply easybuild/easyconfigs/m/manta/manta-1.6.0.eb]: good spot, thx @migueldiascosta fixed  ✔️

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/eef643de50d7858b3d2bd19fc09bb8b4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3115.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5a0126e17c3e343f7d3fd947ed862c7b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/942b963f4925e2d0a2a2984d919541ef for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robqiao!

**Pull request merged**